src/jobs/models/jobManager.js

```diff
@@ -283,8 +289,10 @@ function addCron(jobId, job, cronExpression, configData) {
                 logger.info(`Skipping job with id: ${jobId} as it's currently disabled`);
             } else {
                 const latestDockerImage = await dockerHubConnector.getMostRecentRunnerTag();
-                const runId = Date.now();
-                const jobSpecificPlatformConfig = await createJobRequest(jobId, runId, job, latestDockerImage, configData);
+                const runId = uuid.v4();
```

Comment:
change `runId` to `reportId` and remove alltogether `run_id` as we discussed together

Reply:
done, should I add to README that `run_id` is renamed to `report_id`?